src/components/TeamProgress/TeamProgressPage.tsx

```diff
@@ -69,6 +71,12 @@ export function TeamProgressPage() {
       toast.error(error?.message || 'Failed to get team progress');
       return;
     }
+    const userProgress = response.find((member) => member.email === user?.email);
+    if (userProgress) {
+      const index = response.indexOf(userProgress);
+      response.splice(index, 1);
+      response.unshift(userProgress);
+    }
```

Comment:
Let's not mutate the `response`

```suggestion

    const currentUserProgress = response.find((member) => member.email === user?.email);
    const otherUserProgresses = response.filter(member => member.email !== user?.email);
```